tlog-witness.md

```diff
@@ -1,4 +1,4 @@
-# The Witness Protocol
+# Transparency Log Witnesses
```

Comment:
How about "Transparency Log Witness Protocol"?
Admittedly it's a bit more of a mouthful, but avoids the semantic widening of just "witnesses".

Reply:
Done